windows/install_codeql.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = 'Stop'
+$ProgressPreference = 'SilentlyContinue'
+
+Write-Host -ForegroundColor Green "Installing CodeQL $ENV:CODEQL_VERSION"
+#             https://github.com/github/codeql-cli-binaries/releases/download/v2.10.3/codeql-win64.zip
+$codeqlzip = "https://github.com/github/codeql-cli-binaries/releases/download/v$ENV:CODEQL_VERSION/codeql-win64.zip"
+$out = 'codeql.zip'
+
+Write-Host -ForegroundColor Green "Downloading $codeqlzip to $out"
+
+(New-Object System.Net.WebClient).DownloadFile($codeqlzip, $out)
```

Comment:
Could you add a hash check, same as what we do for other files we download, eg. [here](https://github.com/DataDog/datadog-agent-buildimages/blob/746a71dcf378fb633efb07ffcefb35d510b41d45/windows/install_go.ps1#L13)?

Reply:
AFAICT; since it's a github release download, there's no published hash.  
I could download and hash it myself, but I don't see what security that would add. 
Since the download URL is `https` and github is trusted, that should be sufficient, yes?

Reply:
Agreed on your points. The part that I'd say makes it useful is to make sure what we download doesn't change over time without us knowing - we want to know if what is behind a given a certain URL changes, even if we trust the domain's owner.